gettsim/parameters/eink_st_abzuege.yaml

```diff
@@ -469,6 +469,55 @@ vorsorge_sonstige_aufw_max:
     scalar: 1900
     reference: Art. 1 G. v. 23.07.2009 BGBl. I S.1959.
 
+vorsorgepauschale_mindestanteil:
+  name:
+    de: Mindestanteil an Beiträgen zu GKV und GPV, der abgesetzt wird
+  description: 39b (2) Nr.3 EStG
+  unit: share
+  2009-07-23:
+    scalar: 0.12
+    reference: Art. 1 G. v. 16.07.2009 BGBl. I S. 1959
+
+vorsorgepauschale_kv_max:
+  name:
+    de: Höchsbetrag für Beiträge zu GKV und PKV, der abgesetzt werden kann. Hängt ab von der Steuerklasse
```

Comment:
This sounds rather like a description. Please keep the name attribute short and move explanations to description.

See also #148

Reply:
OK